snap/snapcraft.yaml

```diff
@@ -1,36 +1,56 @@
-name: saber
-version: 0.9.4
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
+name: saber # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.9.4' # just for humans, typically '1.2+git' or '1.3.2'
+summary: A cross-platform libre handwritten notes app # 79 char long summary
+description: |
+  A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and the web!
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+license: GPL-3.0+
+compression: lzo
+architectures:
+  - build-on: amd64
```

Comment:
Please also add `build-for: amd64` ([link to docs](https://snapcraft.io/docs/architectures)) like the Flatpak
https://github.com/adil192/saber/blob/21171cac3d2bcc8d001ccd7d03c0f64a7c367d1f/flatpak/com.adilhanney.saber.json#L22

Note that there is also a shorthand for this (Example 2 on the docs):
```yaml
architectures:
  - amd64
```